tests/docker_ssi/test_docker_ssi.py

```diff
@@ -147,7 +156,10 @@ def test_instrumentation_source_ssi(self):
         assert root_span, f"No traces found for request {self.r.get_rid()}"
         assert "service" in root_span, f"No service name found in root_span: {root_span}"
         # Get all captured telemetry configuration data
-        configurations = interfaces.test_agent.get_telemetry_configurations(root_span["service"], None)
+        configurations = interfaces.test_agent.get_telemetry_configurations(
+            root_span["service"], root_span["meta"]["runtime-id"]
```

Comment:
I have not been able to get it to work without adding the runtime-id. I tested the original PR with the latest tracer and it didn't work.